pinotdb/sqlalchemy.py

```diff
@@ -219,7 +219,7 @@ def get_metadata_from_controller(self, path):
         return result
 
     def get_schema_names(self, connection, **kwargs):
-        return self.get_metadata_from_controller("/schemas")
+        return ["default"]
```

Comment:
So this gives `SELECT xxx FROM default.myTable` instead of  `SELECT xxx FROM myTable.myTable` ?

Reply:
Yes though from my experience using this connector in Superset you don't need the schema at all i.e. `SELECT xxx FROM myTable` just works. So this would just be for things like the schema dropdown UI and creating Superset Datasets (we had a problem where users were creating duplicate Datasets under different schemas e.g. `myTable1.myTable1` and `myTable2.myTable1`